buildbot/osuosl/master/config/builders.py

```diff
@@ -1646,6 +1646,43 @@ def _get_documentation_builders():
              }
            ]
 
+# Builders for ML-driven compiler optimizations.
+def _get_ml_compiler_opt_builders():
+    common_extra_args = ["-DCMAKE_BUILD_TYPE=Release", ]
+    return [
+        {'name': "ml-opt-dev-x86-64",
+         'mergeRequests': False,
+         'slavenames':["ml-opt-dev-x86-64-b1"],
```

Comment:
what is the b1 suffix about?

Reply:
if we later need more bots - it was based on what I saw sanitizers did.